DOCS/man/console.rst

```diff
@@ -175,3 +175,14 @@ Configurable Options
     The ratio of font height to font width.
     Adjusts table width of completion suggestions.
     Values in the range 1.8..2.5 make sense for common monospace fonts.
+
+``select_font_size``
+    Default: 26
```

Comment:
Do we really want another size that user need to adjust? Either way, maybe it should scale with video like stats.lua does? I know there was discussion about it, but I didn't follow it, sorry.

Reply:
> I think this selection should be bigger. Changing all console maybe is not best like #14088, but track selection and things like that should have the same size as F9 for example. It is impossible to see anything if you use it in HTPC setup.

So did you mean it should it should just use `--osd-font-size`? The problem is that the same value in `--script-opts-append=console-font_size=55` or even completely omitting `'\\fs' .. font_size ..` looks much bigger than in the OSD (also in stats.lua) and I don't know how to replicate the same size.

Reply:
Size is not constant, it gets smaller with bigger window and bigger with smaller window.

> So did you mean it should it should just use --osd-font-size?

I mostly mean that I like stats.lua font scaling with window/video.

Reply:
Reverted the new options. I will open an other PR to implement `vidscale` in `console.lua`.